src/Angor/Client/Pages/Invest.razor

```diff
@@ -374,6 +374,7 @@ else
     TransactionInfo? signedTransaction;
     Transaction unSignedTransaction;
     string InvestorPubKey;
+    private ProjectValidationService validationService = new ProjectValidationService();
```

Comment:
you wanna use injection this is not common practice remember we always inject services on startup